LithoUtils/Classes/Base/Models/CodableUIButtonStyle.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CodableUIButton.swift
+//  LithoUtils
+//
+//  Created by Remmington Damper on 1/9/22.
+//
+
+import Foundation
+import LithoOperators
+import Prelude
+import UIKit
+
+open class CodableUIButtonStyle: CodableViewStyle {
+    open var buttonTitleColor: String?
+    open var buttonTitleShadowColor: String?
+    open var buttonTitleAndImageTintColor: String?
+    open var buttonFont: CodableFont?
+}
+
+public func styleButtonFunction(given style: CodableUIButtonStyle) -> (UIButton) -> Void {
+    let doNothing: (UIButton) -> Void = { _ in }
+    var result: (UIButton) -> Void = doNothing
+    
+//    result <>= style.buttonTitleColor |> (~>UIColor.init(hexString:) >>> UIButton.setTitleColor(_:for:))
+//    result <>= style.buttonTitleShadowColor |> (~>UIColor.init(hexString:) >>> (\UIButton.setTitleShadowColor() *-> set))
```

Comment:
Let's either get these working or remove the commented code

Reply:
I actually don't know how to make those work. I need help with using the litho operators. I'm still learning them.